backend/src/routes/v0/tool/info.py

```
@@ -0,0 +1,77 @@
+################################################################################
+@router.post(
+    "/a2a/info",
+    responses={
+        status.HTTP_200_OK: {
+            "description": "All capabilities.",
+            "content": {"application/json": {"example": A2A_GET_AGENT_CARD_EXAMPLE}},
+        }
+    },
+)
+async def get_a2a_agent_card(
+    config: dict[str, A2AServer] = Body(openapi_examples=A2A_DICT_EXAMPLE),
```

Ruff (0.13.3)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)